Dockerfile

```diff
@@ -1,117 +1,119 @@
 FROM ubuntu:18.04 as build-dep
 
-# Use bash for the shell
-SHELL ["bash", "-c"]
+# Use bash for the SHELL
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN echo "Etc/UTC" > /etc/localtime && \
+	apt-get update && \
+	apt-get --no-install-recommends -y install apt-utils wget python ca-certificates
 
 # Install Node v12 (LTS)
 ENV NODE_VER="12.16.1"
 RUN	ARCH= && \
-    dpkgArch="$(dpkg --print-architecture)" && \
-  case "${dpkgArch##*-}" in \
-    amd64) ARCH='x64';; \
-    ppc64el) ARCH='ppc64le';; \
-    s390x) ARCH='s390x';; \
-    arm64) ARCH='arm64';; \
-    armhf) ARCH='armv7l';; \
-    i386) ARCH='x86';; \
-    *) echo "unsupported architecture"; exit 1 ;; \
-  esac && \
-    echo "Etc/UTC" > /etc/localtime && \
-	apt update && \
-	apt -y install wget python && \
+	dpkgArch="$(dpkg --print-architecture)" && \
+	case "${dpkgArch##*-}" in \
+		amd64) ARCH='x64';; \
+		ppc64el) ARCH='ppc64le';; \
+		s390x) ARCH='s390x';; \
+		arm64) ARCH='arm64';; \
+		armhf) ARCH='armv7l';; \
+		i386) ARCH='x86';; \
+		*) echo "unsupported architecture"; exit 1 ;; \
```

Comment:
Erroring out on unsupported architectures was the first thing the Dockerfile did. Now, it'll do a bunch of stuff before that. I think it should still error out as soon as possible.

Reply:
Please test again.